lib/common/migrations/20231115184124_cache.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS cache_config
```

Comment:
-- adds `cache_config` table for storing the config for a publication. `duration` should be time in minutes
-- adds columns `cache_path` and `expires_at` to indicate where in cache the file is located, and when file will be purged